management-api-server/src/main/java/com/datastax/mgmtapi/resources/MetadataResources.java

```diff
@@ -102,4 +102,20 @@ public Response getFeatureSet() {
           return Response.ok(featureSet).build();
         });
   }
+
+  @GET
+  @Path("/clustername")
```

Comment:
Suggestion: In many respects, I would rather have this returned from some endpoint like `/hosts` or `/endpoints` as a part of their response. However, given this is sitting in a v1 API, I'm not going to fuss about it.

Reply:
I wouldn't mind adding to /endpoints as field as it doesn't break anything when new fields appear there (that happens naturally with Cassandra versions). As long as Cassandra itself doesn't add such field name.